commit d7ff93eab23c41a15f7d177ab766bf7e9350bed8 (refs/changes/05/2305/1)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2017-07-31T11:26:44-07:00 (2 years, 2 months ago)
    
    DOCKER-1081 test api-images-v1-v2.test.js failing due to wrong imgapi image layer

diff --git a/test/integration/api-images-v1-v2.test.js b/test/integration/api-images-v1-v2.test.js
index dd4e527..280ac89 100644
--- a/test/integration/api-images-v1-v2.test.js
+++ b/test/integration/api-images-v1-v2.test.js
@@ -230,7 +230,7 @@ test('init docker images', function (tt) {
                     + contentLayers.length));
                 return;
             }
-            var digestChain = manifest.layers.slice(0, layerIdx).map(
+            var digestChain = manifest.layers.slice(0, layerIdx+1).map(
                 function (layer) {
                     return layer.digest;
                 }
